databuilder/models/standalone_column_model.py

```diff
@@ -7,7 +7,10 @@
 from databuilder.models.table_metadata import TableMetadata, DESCRIPTION_NODE_LABEL
 
 
-class TestColumnMetadata(Neo4jCsvSerializable):
+# This class is needed to handle csv based column loading, since the main column model
+# table_metadata.ColumnMetadata requires table_metadata.TableMetadata as well, and this cannot
+# be represented in csv form
+class StandaloneColumnMetadata(Neo4jCsvSerializable):
```

Comment:
I think it will be a bit confusing to have to column metadata model.  I think ideally we should refactor the table metadata model to decouple with column metadata model.  But to unblock, I would prefer to remove the model rename and only commit the csv extractor for this pr. WDYT?

Reply:
I'm not sure. In many ways, I actually think having column and tag coupled to table makes sense. It's also cleaner this way. Really, needing a dedicated model is just a side effect of loading from CSV since CSV is not heirachical. 

Overall, I think file based loading is useful, but, for the cleanest model, it will likely need to be json. But, this would take us some time to get to.

I did leave a comment at the top of the class, is there anything else that can be done to make it less confusing, in the interim?

Reply:
@feng-tao perhaps we could get another opinion if you're not convinced? I would be happy to chat on slack as well, if you think it would help? I'm a bit blocked on this, so, anything I can do to move this along would be appreciated :)

Reply:
cc @jinhyukchang 
Yeah, also I am oncall this week so I may not able to get back you very quickly :(

Reply:
@feng-tao bringing this one back up; @samshuster and I talked thru this Friday. Without moving to json, I'm not sure there is a good way around this. 

The issue is that the table meta object that contains table, column, csv does not lend itself well to csv serialization. But, breaking that class up does not make much sense, since in an ideal use case, a column should be bound to a table and it's tags. 

So, I think having a dedicated stand alone column class makes sense, until we can build out a json loader. 

Can you take another look? Feel free to ping me on slack, I'd be happy to take 15 mins and chat with you, or, think about alternatives.

Reply:
I c, I think we could leave it for now. And revisit on how to consolidate all the models later.

Reply:
Thank you @feng-tao ; do you know when it can get released to pypi? It's not a major blocker,  but, it would be nice.